src/SharpBrick.PoweredUp.BlueGigaBLE/SharpBrick.PoweredUp.BlueGigaBLE.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
use net5.0 or net5.0-windows

Reply:
this might also fix the build

Reply:
After the last commit/push the build (CI in GitHub) worked now. 
I'm not sure what your intention with the C# 9 / .NET 5 dev is; I used now "net5.0-windows10.0.19041.0" for the TargetFramework. Actually only "net5.0" should be enough (the BlueGiga is not dependant on special Windows-libraries like tyour WinRT-implementation is/must be.

Reply:
hm .. are you abstract enough to handle both COM ports and whatever linux device file is used for the dongle. Otherwise, I would enforce the assembly to Windows only despite you are not using a single more library.

Reply:
I have a Raspberry Pi 3 and 4 available and will do some tests with them; maybe I can get my fingers also on PC with a Linux installed; but this will take a while. So it would be better to make it in an additional pull request later

Reply:
This is just for arguments sake of doing this assembly for `net5.0` vs `net5.0-windows`.